sphinx/builders/html.py

```diff
@@ -365,6 +365,28 @@ def default_translator_class(self):
         else:
             return HTMLTranslator
 
+    @property
+    def math_renderer(self):
+        # type: () -> unicode
+        name = self.get_builder_config('math_renderer', 'html')
+        if name is not None:
+            # use given name
+            return name
+        else:
+            # not given: choose a math_renderer from registered ones as possible
+            renderers = list(self.app.registry.html_inline_math_renderers.keys())
+            print(renderers)
+            if len(renderers) == 1:
+                # only default math_renderer (mathjax) is registered
+                return renderers[0]
+            elif len(renderers) == 2:
+                # default and another math_renderer are registered; prior the another
```

Comment:
This is useful if user want to use another math_renderer extension.
```
extensions = ['sphinx.ext.imgmath']
```

Sphinx detects it automatically and enables it as default math renderer.